client/blocks/reader-post-card/docs/example.jsx

```diff
@@ -41,18 +40,19 @@ const searchItems = [
 	}
 ];
 
-const RefreshCards = React.createClass( {
-	displayName: 'RefreshCard',
+const RefreshCards = () => (
+	<div className="design-assets__group">
+		<div>
+			{ searchItems.map( item =>
```

Comment:
Thanks @SpenUK, you're absolutely right. It should have parentheses now it's on multiple lines. Fixed in df26d73.